scipy/optimize/_qap.py

```diff
@@ -43,97 +41,100 @@ def quadratic_assignment(
+                :math:`K`.
+
+                The algorithm will be repeated :math:`k` times
+                from randomized initial search positions
+                :math:`P_0 = (\alpha J + (1- \alpha) K`, where
+                :math:`J` is given by option `init_J`,
+                :math:`\alpha` is given by option `init_weight`,
+                :math:`k` is given by option `init_k`, and
+                :math:`K` is a random doubly stochastic matrix.
+            init_k : int, positive (default = 1)
+                The number of randomized initial search positions :math:`P_0`
+                from which the FAQ algorithm will proceed.
             maxiter : int, positive (default = 30)
-                Integer specifying the max number of Franke-Wolfe iterations.
-                FAQ typically converges with modest number of iterations.
+                Integer specifying the max number of Franke-Wolfe iterations
+                per initial search position. FAQ typically converges within a
+                modest number of iterations.
             shuffle_input : bool (default = True)
                 To avoid artificially high or low matching due to inherent
                 sorting of input adjacency matrices, gives users the option
-                to shuffle the nodes of `cost_matrix`. Results are then
-                unshuffled so that returned `col_ind` matches the node order
-                of inputs.
+                to shuffle the nodes. Results are then unshuffled so that the
+                returned results correspond with the node order of inputs.
             eps : float (default = 0.05)
-                A positive, threshold stopping criteria such that Franke-
-                Wolfe continues to iterate while Frobenius norm of
-                :math:`(P_{i}-P_{i+1}) > eps`, where :math:`i` is the
-                iteration number.
+                A threshold for the stopping criterion. Franke-Wolfe
+                iteration terminates when the change in search position between
+                iterations is sufficiently small, that is, when the Frobenius
+                norm of :math:`(P_{i}-P_{i+1}) \leq eps`, where :math:`i` is
+                the iteration number.
 
     Returns
     -------
     res : OptimizeResult
-        A :class:`scipy.optimize.OptimizeResult` consisting of the fields:
+        A :class:`scipy.optimize.OptimizeResult` containing the following
+        fields.
 
             col_ind : 1-D array
-                An array of column indices corresponding to the optimal
-                permutation (with the fixed seeds given) of the
-                nodes of `dist_matrix`, to best minimize the objective
-                function.
+                An array of column indices corresponding with the best
+                permutation of the nodes of `dist_matrix` found.
             score : float
-                The optimal value of the objective function.
```

Comment:
Is it guaranteed to be locally optimal? Not knowing, I avoided using the word, and said "best found" above.